tools/packages/minion/minion.spec

```diff
@@ -71,7 +71,6 @@ Requires(pre):  jicmp >= 3.0.0
 Requires:       jicmp6 >= 3.0.0
 Requires(pre):  jicmp6 >= 3.0.0
 Provides:	opennms-plugin-api = %{opa_version}
-Recommends:	haveged
```

Comment:
As I recall, `haveged`  was added for entropy gathering purposes, as there was some fairly easily reproducible case where something (Karaf?) would block startup in some environments waiting for sufficient entropy to gather, and `haveged` solved that problem.  https://opennms.atlassian.net/browse/NMS-8959

Reply:
See here: https://github.com/jirka-h/haveged

Reply:
el9 is on 5.14 and everything older is EOL, so I can't imagine there is any reason to keep it around.